standard_rules/brute_force_by_ip.py

```diff
@@ -0,0 +1,14 @@
+import panther_event_type_helpers as event_type
+
+
+def rule(event):
+    # filter events on unified data model field
+    return bool(event.udm('event_type')) and event.udm(
+        'event_type') == event_type.FAILED_LOGIN
```

Comment:
Again this seems simpler as just:

```
return event.udm('event_type') == event_type.FAILED_LOGIN
```

If `event.udm('event_type')` returns a False truthy value, then that will just always evaluate to False.